HelpSource/Classes/TGrains.schelp

```diff
@@ -58,138 +89,154 @@ Examples::
 code::
 
 s.boot;
-b = Buffer.read(s, Platform.resourceDir +/+ "sounds/a11wlk01.wav");
+b = Buffer.read(s, Platform.resourceDir +/+ "sounds/a11wlk01-44_1.aiff");
 
 (
-{
-	var trate, dur, rate;
-	trate = MouseY.kr(2,200,1);
-	dur = 4 / trate;
-	rate = Dseq([10, 1, 1, 0.5, 0.5, 0.2, 0.1], inf);
-	TGrains.ar(2, Impulse.ar(trate), b, rate, MouseX.kr(0,BufDur.kr(b)), dur, Dseq([-1, 1], inf), 0.1, 2);
-}.scope(zoom: 4);
+SynthDef(\TgrainsExample1, {arg buffer, granular;
+	granular = TGrains.ar(
+		numChannels: 2,
+		trigger: Impulse.ar(MouseY.kr(2,200,1)),
+		bufnum: buffer,
+		rate: Dseq([10, 1, 1, 0.5, 0.5, 0.2, 0.1], inf),
+		centerPos: MouseX.kr(0,BufDur.kr(buffer)),
+		dur: 4 / MouseY.kr(2,200,1),
+		pan: Dseq([-1, 1], inf),
+		amp: 0.1,
+		interp: 2);
+	Out.ar(0, granular ! 2);
+}).add;
 )
 
+
 (
-{
-	var trate, dur, clk, pos, pan;
-	trate = MouseY.kr(8,120,1);
-	dur = 12 / trate;
-	clk = Impulse.kr(trate);
-	pos = MouseX.kr(0,BufDur.kr(b)) + TRand.kr(0, 0.01, clk);
-	pan = WhiteNoise.kr(0.6);
-	TGrains.ar(2, clk, b, 1, pos, dur, pan, 0.1);
-}.scope(zoom: 4);
+SynthDef(\TgrainsExample2, {arg buffer, granular; var trate, dur, clk, pos, pan;
+	trate = MouseY.kr(8,120,1); dur = 12 / trate; clk = Impulse.kr(trate);
```

Comment:
For clarity, please keep each variable assignment on a separate line. Do not collapse multiple statements onto one line. In help files, we should err on the side of code readability.